### PR TITLE
Optimize query/search on growing segment while output vector field

### PR DESCRIPTION
**Pull request by yah01**
/kind improvement

### CODE REVIEW

File: internal/core/src/segcore/SegmentGrowingImpl.cpp [hunk 1]
```diff
@@ -426,18 +427,18 @@ SegmentGrowingImpl::bulk_subscript_impl(FieldId field_id,
     auto vec_ptr = dynamic_cast<const ConcurrentVector<T>*>(&vec_raw);
     AssertInfo(vec_ptr, "Pointer of vec_raw is nullptr");
     auto& vec = *vec_ptr;
-    std::vector<uint8_t> empty(element_sizeof, 0);
 
     auto copy_from_chunk = [&]() {
         auto output_base = reinterpret_cast<char*>(output_raw);
         for (int i = 0; i < count; ++i) {
```
Comment (jiaoew1991) [inline internal/core/src/segcore/SegmentGrowingImpl.cpp:433]: Is it possible to change invalid_seg_offset bitset and change for loop to memcpy?🤔

Comment (yah01) [reply internal/core/src/segcore/SegmentGrowingImpl.cpp:433]: it's possible, but we need to check all the paths to filling data to make sure no invalid offsets would be passed, working on it

**Comment by yah01** [comment]
This reduces the CPU and memory usage to create a vector, this would perform much better for large vector field

**Pull request merged**